ui/src/underlaysSlice.ts

```diff
@@ -75,6 +75,7 @@ export type CohortReviewConfig = {
 export type CohortReviewAttribute = {
   title: string;
   key: string;
+  labelOnlyBoolean?: boolean;
```

Comment:
Don't really like the name `labelOnlyBoolean` but hard to indicate exactly what the property does wihtout making it super long. Let me know if anyone has a better suggestion.

Reply:
I would suggest: "showLabel"  or "showConditional" or "showIfTrue"